src/Context/Product-Context.js

```diff
@@ -0,0 +1,41 @@
+
+import React, { createContext, useState, useEffect, useContext } from "react"
+import { useAxios } from "../customHooks/useAxios"
+
+const ProductContext = createContext()
+
+const ProductProvider = ({ children }) => {
+    const [productsFromDB, setProductsFromDB] = useState([])
+    const [categoriesFromDB, setCategoriesFromDB] = useState([])
+    const { response, loading, operation: fetchProducts } = useAxios()
+    const { response: responseFromCategories, operation: fetchCategories } = useAxios()
```

Comment:
Try to follow a pattern for consistency and easy debugging.
You can also create an alias for `response` as `responseProducts` and `loading` as `responseLoading`